### DISCUSSION TITLE
Pulls merge API - allow an admin privilege merge to bypass branch protections

### DISCUSSION THREAD

**Discussion by sevagh** [community/community#9829] [Pull Requests]
Hello,

In the Pulls Merge API: https://docs.github.com/en/rest/reference/pulls#merge-a-pull-request

There is no option for the "admin" merge which ignores required status checks or other branch protections.

Such an admin option exists already in the UI, and we can see that the `--admin` flag was added to the `gh` cli for the same purpose:

https://github.com/cli/cli/issues/1198
https://github.com/cli/cli/pull/4071/files

Could there be a parameter added to the pulls merge API to add `admin: true` or something similar?

Otherwise, are there any existing ways to achieve this goal?

Thanks.

**Comment by daveygit2050** [comment]
It appears you do not need to specify any additional options when merging via the API as an admin. It will just work.
